assets/src/edit-story/components/helpCenter/useHelpCenter/effects.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import { TIPS } from '../constants';
+
+const TIP_KEYS_MAP = Object.keys(TIPS).reduce((keyMap, key) => {
+  keyMap[key] = true;
+  return keyMap;
+}, {});
```

Comment:
💭 probably not necessary atm since it's only used here, but maybe this should live in `constants.js` 🤷

Reply:
Yea good call. I'm working in this file in my next PR, I'll move it there then